EIPS/eip-42.md

```diff
@@ -0,0 +1,399 @@
+---
+eip: 42 (final number to be confirmed)
+title: T-REX - Token for Regulated EXchanges
+authors: Joachim Lebrun, Tony Malghem, Kevin Thizy, Luc Falempin, Adam Boudjemaa
```

Comment:
```suggestion
authors: Joachim Lebrun (@Joachim-Lebrun), Tony Malghem, Kevin Thizy, Luc Falempin, Adam Boudjemaa
```

Reply:
Please fill in other author's github handles.

Reply:
please add a `discussions-to: ` link (usually to a thread on https://ethereum-magicians.org) per eip-1

Reply:
discussion created here [EIP-3643 : Proposition of the T-REX token standard for securities](https://ethereum-magicians.org/t/eip-3643-proposition-of-the-t-rex-token-standard-for-securities/6844) i will update on the PR accordingly